tools/image_modifier.py

```diff
@@ -30,23 +31,23 @@ def GetDefinition(filename):
 def CreateSmallVersion(filename):
     small_filename = DetermineSmallFilename(filename)
     big_image = Image.open(filename)
-    resized = big_image.resize((34, int(GetScalingFactor(big_image.size[0]) * big_image.size[1])))
+    resized = big_image.resize((34, int(GetScalingFactor(big_image.size[0]) * big_image.size[1]) + 6), Image.Resampling.BICUBIC)
     new = Image.new("RGBA", (65, 67))
-    new.paste(resized, (15,int((67-resized.size[1])/2)))
-    rotated = new.rotate(5)
+    new.paste(resized, (15,int((67-resized.size[1])/2) + 2))
+    rotated = new.rotate(5, Image.Resampling.BICUBIC)
     on_canvas = Image.open("data/resources/images/nat_ideas_template_canvas.png")
-    on_canvas.paste(rotated,(-6,-4),rotated)
+    on_canvas.alpha_composite(rotated,(-6,-4))
     frame = Image.open("data/resources/images/nat_ideas_template_frame.png")
-    on_canvas.paste(frame, (0,0), frame)
+    on_canvas.alpha_composite(frame)
     on_canvas.save(small_filename)
 
 def UpdateMappings(filename):
```

Comment:
You can actually get rid of UpdateMappings, we're going to do things slightly differently this time around.

Reply:
Ok, that function call is removed. Also removed the unnecessary ``sys`` import